src/renderer/scss/component/_card.scss

```diff
@@ -4,7 +4,7 @@
   border-radius: var(--card-radius);
   user-select: text;
   display: flex;
-  position: relative;
+  position: initial;
```

Comment:
maybe if you just remove this line should also work?

Reply:
That would be ideal if it works.

Reply:
@seanyesmunt yes it works

Reply:
![image](https://user-images.githubusercontent.com/26609573/46098889-90791900-c1ef-11e8-8eec-7b8ab663dcac.png)